### ISSUE TITLE
A11y_AzureDataStudio_Dashboard_ScreenReader: ORCA Screen reader is announcing "Learn More" link as button.

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio#22690]
"[Check out Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### #GitHub Tags:
#A11yMAS; #A11ySev3; #A11yTCS; #DesktopApp; #SH-AzureDataStudio-Linux-Jul2021; #Azure Data Studio; #WCAG4.1.2; #Linux; #ScreenReader; #ORCA;

### Environment Details:​
Application Name: MySQL extension for Azure Data Studio
Application Version: 1.42.0-insider
Operating System: Ubuntu 20.04.2 LTS
GNOME Version: 3.36.8
ORCA: 42.2-1ubuntu1
Distributor ID: Ubuntu
Description: Ubuntu 22.10
Release: 22.10
Codename: kinetic

### Repro Steps:

1. Launch Azure Data Studio application.
2. TAB to "Servers" and select any server from the list and hit ENTER on 'Connect' button.
3. TAB to connected server and right click on it and TAB to "Manage" and hit ENTER key.
4. Start the ORCA screen reader. TAB to "Learn more" link.
5. Observe that ORCA Screen reader is announcing "Learn More" link as button.

### Actual Result:​
ORCA Screen reader is announcing "Learn More" link as button.

### Expected Result:​
ORCA Screen reader should be announcing "Learn More" link as "Learn More link."

### User Impact:
Screen reader users will get confused about the role of the "Learn More", as after selecting link is opening but screen reader announcing it as button.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Attachment:

https://user-images.githubusercontent.com/90901519/228772351-cdb54578-2813-475d-8273-495f2a8219a8.MOV

**Comment by alanrenmsft** [comment]
@PoojaNamde  this is by design, it is indeed a button but clicking on it will cause a window to be opened. isn't this common?

**Comment by PoojaNamde** [comment]
@alanrenmsft 

When we click on "Learn more" link, new page is getting opened in browser. It means it's a link and the role should be provided as "Link". This issue should be fixed.
